client/app/components/dashboards/gridstack/gridstack.js

```diff
@@ -0,0 +1,90 @@
+import $ from 'jquery';
+import _ from 'underscore';
+import 'jquery-ui/ui/widgets/draggable';
+import 'jquery-ui/ui/widgets/droppable';
+import 'jquery-ui/ui/widgets/resizable';
+import 'gridstack/dist/gridstack.css';
+
+window.$ = window.jQuery = $;
+window._ = _;
```

Comment:
Isn't this handled by the `webpack.ProvidePlugin` (which we already use for jQuery)?

Reply:
yes, probably accidentally missed it. but thanks for pointing